### ISSUE TITLE
Yara aborts on high occurence numbers

### ISSUE THREAD

**Issue by jvoisin** [plusvic/yara#288]
```
rule OneLiner
{
    strings:
        $oneliner = /[^\n]{500}/
    condition:
      any of them
}
```

This rule file produce this result:

```
$ yara -r rules .
yara: re.c:1117: yr_re_emit_code: Assertion `total_size < 16384' failed.
zsh: abort (core dumped)  yara -r .
```

**Comment by axqd** [comment]
Thanks. I just noticed, except for exit() calls, there are also abort() from assert(). Now not sure how to deal with that. I can disable the whole Yara feature instead of crashing the daemon...

**Comment by plusvic** [comment]
Fixed in b1106adfa8f891711309f5cb71333aacebcef62a. Large regular expressions cause an error instead crashing the program.
